src/stats/actions.ts

```diff
@@ -264,20 +282,21 @@ function handleActionCompute(state: PlayerActionState, indices: PlayerIndexedTyp
     incrementCount("lCancelCount.fail", playerFrame.lCancelStatus === 2);
   }
 
+  // check for edge canceled aerial
   if (
     isAerialLanding(prevAnimation) &&
     (currentAnimation === State.FALL || currentAnimation === State.TEETER) &&
-    playerFrame.internalCharacterId
+    playerFrame.internalCharacterId !== null
```

Comment:
Maybe you should use this instead: https://github.com/project-slippi/slippi-js/blob/5a46672b49499d1b650d4d79a6250162b1a5b981/src/utils/exists.ts

I think you probably dont want undefined either.